checks.d/rabbitmq.py

```diff
@@ -198,7 +199,10 @@ def get_stats(self, instance, base_url, object_type, max_detailed, filters, auth
 
                 match_found = False
                 for p in regex_filters:
-                    if re.search(p, name):
+                    match = re.search(p, name)
+                    if match:
+                        if instance.get("tag_families", False) and match.groups():
```

Comment:
For boolean options, could you use (instead of `instance.get("tag_families", False)`):

```python
_is_affirmative(instance.get("tag_families", False))
```

after having imported `_is_affirmative`:

```python
from config import _is_affirmative
```

(you can find an example of this usage in the elasticsearch check: https://github.com/stripe/dd-agent/blob/joshu_add-rabbitmq-family-tagging/checks.d/elastic.py#L254)